src/lib/DeployUtil.ts

```diff
@@ -746,10 +746,7 @@ export class ExecutableDeployItem implements ToBytes {
     if (target instanceof CLURef) {
       runtimeArgs.insert('target', target);
     } else if (target instanceof CLPublicKey) {
-      runtimeArgs.insert(
-        'target',
-        CLValueBuilder.byteArray(target.toAccountHash())
-      );
```

Comment:
It should still support using an account hash. EE will accept that as a target.

Reply:
for the end user it is transparent, they still need to provide `CLUref` or `CLPublicKey` as a target (see line 737).
this is only internal change.

Reply:
I see, that's the problem then - Rust library allows for using `AccountHash` type as target. If it's not changing here then that's a separate issue that we don't have to address here.